Renderer.js

```diff
@@ -0,0 +1,53 @@
+
+class Renderer {
+   constructor(data) {
```

Comment:
I wouldn't get the data here in the constructor. You should pass the data to the `mainRender` method. If you do it like this you won't. need to create a new instance of `Renderer` every time you want to render.